client/flags/flags.go

```diff
@@ -148,6 +149,7 @@ func RegisterRestServerFlags(cmd *cobra.Command) *cobra.Command {
 	cmd.Flags().Uint(FlagMaxOpenConnections, 1000, "The number of maximum open connections")
 	cmd.Flags().Uint(FlagRPCReadTimeout, 10, "The RPC read timeout (in seconds)")
 	cmd.Flags().Uint(FlagRPCWriteTimeout, 10, "The RPC write timeout (in seconds)")
+	cmd.Flags().Int64(FlagRPCMaxBodyBytes, 1000000, "The RPC max body bytes (in MB)")
```

Comment:
Maybe better use `Uint` (or `Uint64`)? I think the flag would validate the input for us

Reply:
Sure, although this should be handled in the underlying handler -- not here.